doc/rados/operations/crush-map.rst

```diff
@@ -1013,10 +1012,10 @@ Migration impact:
    of data movement.  Use caution on a cluster that is already
    populated with data.
 
-firefly
--------
+firefly (CRUSH_TUNABLES3)
+-------------------------
 
-The firefly tunable profile (CRUSH_TUNABLES2 feature) fixes a problem
+The firefly tunable profile fixes a problem
```

Comment:
Are you sure on this point? According to this page https://access.redhat.com/documentation/en/red-hat-ceph-storage/1.3/storage-strategies/chapter-9-crush-tunables it's TUNABLES3 (which I have added in the title), no?